src/schemas/execute.yaml

```diff
@@ -0,0 +1,121 @@
+BatchCallArgs:
+  title: Arguments for batch call
+  type: object
+  required:
+    - block
+    - calls
+  properties:
+    block:
+      title: Block tag
+      $ref: '#/components/schemas/BlockNumberOrTag'
+    stateOverrides:
+      title: State overrides
+      $ref: '#/components/schemas/StateOverrides'
+    calls:
+      title: List of calls
+      $ref: '#/components/schemas/Calls'
```

Comment:
This should also have an optional block producer address, and there should be a standardized default value across clients.

Reply:
This is settable in the block overrides set.